packages/web/src/components/createCell.tsx

```diff
@@ -169,6 +169,9 @@ export function createCell<CellProps = any>({
           } else if (loading) {
             return <Loading {...queryRest} {...props} />
           } else {
+            console.warn(
+              'Check for debug log from Apollo, which might help explain the following error.'
```

Comment:
```suggestion
              `If you're using Apollo Client, check for its debug logs here in the console, which may help explain the error.`
```

Reply:
I don't mind backticks or double quotes, just had to use one or the other after adding `you're`.